src/resources/postcss/views/_list.pcss

```diff
@@ -101,13 +101,7 @@
 	}
 
 	.tribe-events-calendar-list__event-title-link {
-		border-bottom: 2px solid transparent;
-		transition: var(--transition-border-color);
-
-		&:focus,
-		&:hover {
-			border-bottom: 2px solid var(--color-text-primary);
-		}
+		@mixin anchor-default;
```

Comment:
same as https://github.com/moderntribe/the-events-calendar/pull/2564#discussion_r295138902